token-swap/js/client/token-swap.js

```diff
@@ -342,54 +359,21 @@ export class TokenSwap {
     return tokenSwap;
   }
 
-  /**
-   * Retrieve tokenSwap information
-   */
-  async getInfo(): Promise<TokenSwapInfo> {
-    const accountInfo = await this.connection.getAccountInfo(this.tokenSwap);
-    if (accountInfo === null) {
-      throw new Error('Failed to find token swap account');
-    }
-    if (!accountInfo.owner.equals(this.swapProgramId)) {
-      throw new Error(
-        `Invalid token swap owner: ${JSON.stringify(accountInfo.owner)}`,
-      );
-    }
-
-    const data = Buffer.from(accountInfo.data);
-    const tokenSwapInfo = TokenSwapLayout.decode(data);
-    if (!tokenSwapInfo.isInitialized) {
-      throw new Error(`Invalid token swap state`);
-    }
-    // already properly filled in
-    // tokenSwapInfo.nonce = tokenSwapInfo.nonce;
-    tokenSwapInfo.tokenAccountA = new PublicKey(tokenSwapInfo.tokenAccountA);
-    tokenSwapInfo.tokenAccountB = new PublicKey(tokenSwapInfo.tokenAccountB);
-    tokenSwapInfo.tokenPool = new PublicKey(tokenSwapInfo.tokenPool);
-    tokenSwapInfo.feesNumerator = Numberu64.fromBuffer(
-      tokenSwapInfo.feesNumerator,
-    );
-    tokenSwapInfo.feesDenominator = Numberu64.fromBuffer(
-      tokenSwapInfo.feesDenominator,
-    );
-    tokenSwapInfo.feeRatio =
-      tokenSwapInfo.feesNumerator.toNumber() /
-      tokenSwapInfo.feesDenominator.toNumber();
-
-    return tokenSwapInfo;
-  }
-
   /**
    * Swap token A for token B
    *
-   * @param source Source token account
-   * @param destination Destination token account
+   * @param userSource User's source token account
+   * @param poolSource Pool's source token account
+   * @param userSource User's destination token account
+   * @param poolSource Pool's destination token account
```

Comment:
Comments here reference the wrong variables `userSource` -> `userDestination` and `poolSource` -> `poolDestination`